pilot/pkg/config/clusterregistry/secretcontroller.go

```diff
@@ -41,6 +42,23 @@ const (
 	maxRetries = 5
 )
 
+var (
+	clusterCreationCounts = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "k8s_cluster_created_count",
```

Comment:
these should be prefixed by `pilot_` to match the other metrics from pilot.

Reply:
Also, please use prometheus naming conventions: https://prometheus.io/docs/practices/naming/#metric-names.

These should be named something like: `pilot_k8s_clusters_created_total` and `pilot_k8s_clusters_deleted_total`.

Reply:
Or, maybe `pilot_k8s_clusters_total` with label `action`: `{created | deleted}`

Reply:
Thanks, addressed